plugins/sentinelone/komand_sentinelone/actions/run_remote_script/action.py

```diff
@@ -0,0 +1,69 @@
+import insightconnect_plugin_runtime
+from insightconnect_plugin_runtime.exceptions import PluginException
+from .schema import RunRemoteScriptInput, RunRemoteScriptOutput, Input, Output, Component
+
+# Custom imports below
+
+
+class RunRemoteScript(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="run_remote_script",
+            description=Component.DESCRIPTION,
+            input=RunRemoteScriptInput(),
+            output=RunRemoteScriptOutput(),
+        )
+
+    def run(self, params={}):
+        agent_ids = params.get(Input.IDS)
+        script_id = params.get(Input.SCRIPT_ID)
+        if not script_id:
+            raise PluginException(
+                cause=f"No script id provided to execute.",
```

Comment:
We need to get rid of all of these f-strings and replace them with regular strings if we're not putting variables into them as they pose security risks and are breaking the checks

Reply:
Thanks @cmcnally-r7 , I'd just noticed the failure in the static code analysis checks and updated them so hopefully this is now resolved.